.golangci.yaml

```diff
@@ -1,3 +1,5 @@
+issues:
+  exclude-use-default: false
```

Comment:
Wow. That is a terribly named option for what it does. I would have no idea looking at the config file. Thanks for explaining in the commit message.